src/frontends/onnx/frontend/src/op/com.microsoft/fusedgemm.cpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "fusedgemm.hpp"
+
+#include <memory>
+
+#include "default_opset.hpp"
+#include "ngraph/builder/reshape.hpp"
+#include "ngraph/op/add.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/op/matmul.hpp"
+#include "ngraph/op/multiply.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector fusedgemm(const Node& node) {
+    OutputVector inputs{node.get_ng_inputs()};
+    Output<ngraph::Node> input_a = inputs.at(0);
+    Output<ngraph::Node> input_b = inputs.at(1);
+    Output<ngraph::Node> input_c;
+
+    if (inputs.size() == 3) {
+        input_c = inputs.at(2);
+    } else {
+        input_c = default_opset::Constant::create(input_b.get_element_type(), ngraph::Shape{}, {0});
+    }
+
+    const auto alpha = node.get_attribute_value<float>("alpha", 1);
+    const auto beta_node = node.get_attribute_as_constant<float>("beta", 1, input_c.get_element_type());
+
+    const bool trans_a = node.get_attribute_value<int64_t>("transA", 0);
+    const bool trans_b = node.get_attribute_value<int64_t>("transB", 0);
+
+    if (trans_a) {
+        input_a = ngraph::builder::opset1::transpose(input_a);
+    }
+
+    if (trans_b) {
+        input_b = ngraph::builder::opset1::transpose(input_b);
+    }
+
+    input_a = ngraph::builder::opset1::flatten(input_a, 1);
+    input_b = ngraph::builder::opset1::flatten(input_b, 1);
+
+    std::shared_ptr<ngraph::Node> matmul_node = std::make_shared<default_opset::MatMul>(input_a, input_b);
+
+    if (alpha != 1) {
+        const auto alpha_node = default_opset::Constant::create(input_b.get_element_type(), Shape{}, {alpha});
+        matmul_node = std::make_shared<default_opset::Multiply>(matmul_node, alpha_node);
+    }
+
+    auto beta_times_input_c = std::make_shared<default_opset::Multiply>(beta_node, input_c);
+
+    auto gemm_res = std::make_shared<default_opset::Add>(matmul_node, beta_times_input_c);
+
+    const auto activation_type = node.get_attribute_value<std::string>("activation", "Relu");
+    if(activation_type=="LeakyRelu") {
+        double activation_alpha = node.get_attribute_value<double>("activation_alpha", 0.01);
+	std::shared_ptr<ngraph::Node> activation_alpha_node = default_opset::Constant::create(input_c.get_element_type(), Shape{1}, {activation_alpha});
+        return {std::make_shared<default_opset::PRelu>(gemm_res, activation_alpha_node)};
+    }
+    return {std::make_shared<default_opset::Relu>(gemm_res)};
+}
+
+}  // namespace set_1
+
+
+namespace set_6 {
+OutputVector fusedgemm(const Node& node) {
```

Comment:
It's not exposed elsewhere (fusedgemm.hpp, ops_bridge.cpp).
It looks very similar to `set_1::fusedgemm`

Reply:
@a3213105 Could you explain the difference between `fusedgemm` from `set_1` and `set_6`?

You have registered only set_1 version in the ops_bridge.cpp:
`REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "FusedGemm", 1, fusedgemm);`

Reply:
My code is based on src/frontends/onnx/frontend/src/op/gemm.cpp. It has set_1 and set_6. I didn't analyze this piece of code carefully before, and it really wasn't necessary to use set_6. I'm going to use part of the code for set_6 instead of set_1 as @mitruska  said in [#11878 (comment).](https://github.com/openvinotoolkit/openvino/pull/11878#discussion_r896675446) .